### ISSUE TITLE
quotemeta_bytes() (and friends) don’t quotemeta anymore!

### ISSUE THREAD

**Issue by drmuey** [drmuey/p5-String-UnicodeUTF8#1]
e.g. 

`say quotemeta_bytes("[foo] bar™");` should be  `"\[foo\] bar™"`

**Comment by drmuey** [comment]
derf

```
-     $s =~ s{([^!#&()*+,\-.\/0123456789:;<=>?ABCDEFGHIJKLMNOPQRSTUVWXYZ\[\]\^_`abcdefghijklmnopqrstuvwxyz{|}~ ])}
+     $s =~ s{([^A-Za-z0-9])}
```

**Comment by drmuey** [comment]
Fixed in v0.23 just uploaded to CPAN
